### DISCUSSION TITLE
Add SVGR

### DISCUSSION THREAD

**Discussion by andrecasal** [epicweb-dev/epic-stack#166] [Ideas]
[SVGR](https://react-svgr.com/) converts SVGs into React components so we can style them at runtime.

This doesn't work at the moment, but with SVGR it would:
```tsx
import Star from '~/icons/star.svg'

const Example = () => (
  <div>
    <Star />
  </div>
)
```

I'd be happy to create a PR for this.

**Comment by kentcdodds** [comment]
I've used SVGR before. I have mixed feelings about putting SVGs directly into React components. There are trade-offs, but I feel like a lot of the time it's sub-optimal. I want to balance simple with optimal though, so I need to explore some of the trade-offs.

If someone wanted to help, creating a couple different examples of different approaches (or finding an existing analysis) would be very helpful.
